.github/workflows/ci.yml

```diff
@@ -19,172 +19,318 @@ jobs:
       - run: git remote add upstream https://github.com/modin-project/modin.git
       - run: git fetch upstream
       - run: npx commitlint --from upstream/master --to HEAD --verbose
+
   lint-black:
     name: lint (black)
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```

Comment:
why bump?

Reply:
Just updating to the next major version.